server-src/services.js

```diff
@@ -44,7 +44,15 @@ async function initServices(app, done) {
 
   const redirects = ['/authenticated', '/prototypes', '/preferences', '/confirmed', '/labelImage', '/selected', '/detected', '/login', '/register', '/v2/wildfirecheck'];
   redirects.forEach(redirectUrl => {
-    app.get(redirectUrl, (req,res) => {res.redirect('/wildfirecheck/?redirect=' + redirectUrl);});
+    app.get(redirectUrl, (req,res) => {
+      let params = [];
+      Object.keys(req.query).forEach(key => {
+        params.push(`${key}=${req.query[key]}`)
+      });
+      const paramsStr = params.length ? ('?' + params.join('&')) : '';
+      let newUrl = '/wildfirecheck/?redirect=' + redirectUrl + encodeURIComponent(paramsStr);
+      res.redirect(newUrl);
+    });
```

Comment:
Stylistically, I’d lean into the functors and use string templates rather than concatenation.

```js
    app.get(url, ({query}, res) => {
      const params = Object.keys(query).map((k) => `${k}=${query[k]}`).join('&');
      let redirectUrl = 
        `/wildfirecheck/?redirect=${url}${params && encodeURIComponent(`?${params}`)}`;

      res.redirect(redirectUrl);
    });
```

Otherwise, LGTM